tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -1382,3 +1382,9 @@ test_i2c_lsm9ds1_mag: lsm9ds1_mag@b9 {
 	compatible = "st,lsm9ds1_mag";
 	reg = <0xb9>;
 };
+
+test_i2c_mb7040: mb7040@b10 {
+	compatible = "maxbotix,mb7040";
+	reg = <0x10>;
+	status-gpios = <&test_gpio 0 0>;
+};
```

Comment:
```suggestion
test_i2c_mb7040: mb7040@ba {
	compatible = "maxbotix,mb7040";
	reg = <0xba>;
	status-gpios = <&test_gpio 0 0>;
};
```

Reply:
Please don't mark comment as resolved please unless you applied the change verbatim AND verified you pushed it. Here you forgot to push your changes, it seems.